src/deps_info.json

```diff
@@ -123,7 +123,7 @@
   "setjmp": ["setThrew", "realloc", "testSetjmp", "saveSetjmp"],
   "longjmp": ["setThrew", "realloc", "testSetjmp", "saveSetjmp"],
   "siglongjmp": ["setThrew", "realloc", "testSetjmp", "saveSetjmp"],
-  "emscripten_longjmp_jmpbuf": ["setThrew", "realloc", "testSetjmp", "saveSetjmp"],
+  "emscripten_longjmp": ["setThrew", "realloc", "testSetjmp", "saveSetjmp"],
```

Comment:
to be sure I understand, now the LLVM backend will never emit `emscripten_longjmp_jmpbuf` any more, but it does emit `emscripten_longjmp`?

Reply:
Yes.